awsDetonationLab.template

```diff
@@ -3061,6 +3017,48 @@
           "Ref": "wazuhUser"
         }
       }
+    },
+    "KinesisToLambda": {
```

Comment:
Is this change necessary in this PR?

Reply:
I am pretty sure this change is just the lines being moved due to the actual changes